server/sonar-server/src/test/java/org/sonar/server/rule/ws/SearchActionMediumTest.java

```diff
@@ -496,16 +498,13 @@ public void get_note_as_markdown_and_html() throws Exception {
     result.assertJson(this.getClass(), "get_note_as_markdown_and_html.json");
   }
 
+  @Ignore
   @Test
   public void filter_by_tags() throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)